bin/make-firefox-bundle

```diff
@@ -28,11 +28,11 @@ fi
     fi
 );
 
-TARGET=firefox-panel node_modules/.bin/webpack
+TARGET=firefox-panel ./node_modules/.bin/build-bundle
 echo "// Generated from: $REV\n" | cat - public/build/bundle.js > "$DEBUGGER_PATH/bundle.js"
-cp public/build/source-map-worker.js "$DEBUGGER_PATH"
-cp public/build/pretty-print-worker.js "$DEBUGGER_PATH"
-cp public/build/styles.css "$DEBUGGER_PATH"
+cp /public/build/source-map-worker.js "$DEBUGGER_PATH"
+cp /public/build/pretty-print-worker.js "$DEBUGGER_PATH"
+cp /public/build/styles.css "$DEBUGGER_PATH"
```

Comment:
I don't understand these changes. Sure we're not creating a top-level `public` folder on the system?

I don't know how I feel above hiding webpack inside of `build-bundle`. The toolbox should not own our webpack build process. We should just require the toolbox in our `main` file, and our webpack will bundle it in, and the project can document whatever babel plugins/etc it needs enabled.